### PR TITLE
Fix up experience for firefox and add svg images

### PR DESCRIPTION
**Pull request by darcyclarke**
/cc @jeffjewiss @rickycodes @brenna @richgilbank

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -301,7 +358,9 @@ <h2 id="mentors">Our Mentors</h2>
     </main>
 
     <script type="text/javascript">
-      particlesJS.load('particles', 'js/particles.json');
+      if ( navigator.userAgent.toLowerCase().indexOf('firefox') === -1 ) {
```
Comment (jeffjewiss) [inline index.html:361]: :cry:

Comment (darcyclarke) [inline index.html:361]: Things got too real...

Comment (vladikoff) [inline index.html:361]: @miketaylr :police_car:

Comment (darcyclarke) [inline index.html:361]: @vladikoff Callin' the cops I see?! You're dead to me vlad!!

Comment (miketaylr) [inline index.html:361]: :cop: So what's the deal? Is there a known bug w/ particlesJS?

**Pull request merged**